app/src/main/java/com/infomaniak/mail/ui/MainActivity.kt

```diff
@@ -77,7 +77,6 @@ import javax.inject.Inject
 @AndroidEntryPoint
 class MainActivity : BaseActivity() {
 
-    // This binding is not private because it's used in ThreadListFragment (`(activity as? MainActivity)?.binding`)
```

Comment:
binding is still public, why remove the comment?

Reply:
This comment dates back to when it was the 1st public binding.
Now there is like 4 or 5 public bindings, so it's a normal behavior.
Hence the comment loses its interest.

Reply:
In fact, this binding is now private : https://github.com/Infomaniak/android-kMail/pull/1606